test/local_creds_test.go

```diff
@@ -89,10 +89,7 @@ func testLocalCredsE2ESucceed(network, address string) error {
 
 	switch network {
 	case "unix":
-		cc, err = grpc.Dial(lisAddr, grpc.WithTransportCredentials(local.NewCredentials()), grpc.WithContextDialer(
-			func(_ context.Context, addr string) (net.Conn, error) {
-				return net.Dial("unix", addr)
-			}))
+		cc, err = grpc.NewClient("passthrough:///"+lisAddr, grpc.WithTransportCredentials(local.NewCredentials()))
```

Comment:
You need to keep the `grpc.WithContextDialer` dial option.

Reply:
done